### DISCUSSION TITLE
Profile: 500 internal server error

### DISCUSSION THREAD

**Discussion by Collins-Omariba** [EddieHubCommunity/LinkFree#4362] [Q&A] [answered]
I get a 500 internal server error when I visit my profile, any ideas why?

**Comment by Collins-Omariba** [comment]
https://linkfree.eddiehub.io/Collins-Omariba

**Reply by eddiejaoude** [reply]
It is now working https://linkfree.eddiehub.io/Collins-Omariba

**Comment by eddiejaoude** [comment, marked as answer]
There is a missing `icon` on this link ....

```js
{
      "name": "portfolio website",

      "url": "https://collins-omariba.github.io"
    },
```

I think you found a bug as it should not break the whole page and use the fallback icon, thank you for raising 👍 

I will created an issue from this discussion

**Reply by eddiejaoude** [reply]
Issue created here https://github.com/EddieHubCommunity/LinkFree/issues/4393)

**Comment by Collins-Omariba** [comment]
Thanks Eddie.
